docker/Dockerfile.staging

```diff
@@ -0,0 +1,22 @@
+FROM node:14.19.0-slim as builder
+
+RUN apt update && apt install cpio -y
+
+COPY package.json yarn.lock* ./
+RUN yarn install
+
+COPY . .
+
+COPY localhost.env .env
+RUN NODE_ENV=production yarn build
+
+FROM node:14-alpine
+
+WORKDIR /app
+COPY --from=builder package.json tsconfig.json yarn.lock ./
+COPY --from=builder build/ ./build
+COPY --from=builder node_modules/ ./node_modules
+
+EXPOSE 3001 3011 3012
+
+ENTRYPOINT [ "yarn", "start" ]
```

Comment:
Do we really need two same files?

Reply:
We should have two different Dockerfiles . one for staging and one for production . because we are using Vault for our production and I have to change the production Dockerfile based on our Vault configuration in the future . (we should speak about off-chain ENV and get to know that which ones is secret and should be put on the vault) 
So , At this moment they are the same . but in the future this Dockerfiles will be totally different .

Reply:
I will add some of our future changes into the production Dockerfile . (for example : we should remove executable permission from it )